src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
   <bitmap name="OccupancyBitmap" type="bitmap8">
     <cluster code="0x0406"/>
+    <cluster code="0x0201"/>
```

Comment:
So this is weird in the spec: normally one cluster cannot reference a data type from another cluster (because a data type name is _not_ a unique identifier for a data type, except for the ones defined in core Data Model; it's always an implicit (cluster, type name) pair with only the second component listed.

This needs to get sorted out in the spec.

Reply:
Cloned into thermostat-server.xml and removed the cluster reference.